### PR TITLE
ocm: add pr-image-mirror to search

### PR DESCRIPTION
**Pull request by SherinV**
remove disabled tag from 2.3
change e2e for search operator

### CODE REVIEW

File: ci-operator/config/open-cluster-management/search-ui/open-cluster-management-search-ui-main.yaml [hunk 1]
```diff
@@ -25,6 +25,14 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: pr-image-mirror
```
Comment (SherinV) [inline ci-operator/config/open-cluster-management/search-ui/open-cluster-management-search-ui-main.yaml:28]: @zlayne FYI: Adding config to push PR images to quay.

**Comment by SherinV** [comment]
@zlayne FYI: Added PR image push config to search-ui.

**Review by xrajesh — APPROVED** [review]

**Review by Anxhela21 — APPROVED** [review]

**Review by zlayne — APPROVED** [review]

**Comment by mqhackett** [comment]
Looks good to me. @SherinV were you able to find and pull the PR image from quay?

**Comment by SherinV** [comment]
@mqhackett I had tested it on insights-metrics. Now, it should be tested along with all the other Prow steps when empty PRs are made across all repos.

**Pull request merged**